iron-image.html

```diff
@@ -355,21 +354,16 @@
         this.style.height = isNaN(this.height) ? this.height : this.height + 'px';
       },
 
-      _preventLoadChanged: function() {
-        if (this.preventLoad || this.loaded) return;
-
-        this._reset();
```

Comment:
Yup! This would have introduced a bug where toggling `preventLoad` from `false` -> `true` would reload your image even if it was already loaded. I'll fix it and add a test.